src/puppetlabs/puppetdb/command.clj

```diff
@@ -450,9 +443,11 @@
   (init [this context]
     (let [response-chan (async/chan 1000)
           response-mult (async/mult response-chan)
-          response-chan-for-pub (async/chan)]
+          response-chan-for-pub (async/chan)
+          concurrent-writes (get-in (get-config) [:command-processing :concurrent-writes])]
```

Comment:
I'm thinking this will go away once we merge into master. There were some questions around how many concurrent writes we should allow, and what would be the optimal value. @wkalt's research has found it doesn't have a huge impact on performance. My thinking now is that we don't constrain writes and just allow the jetty threads to write without issue